### PR TITLE
add ci: surely this will run the first time with no errors

### PR DESCRIPTION
**Pull request by bendhouseart**
Just creating the draft PR to see magic happen in actions, will notify with '@'s once ready for review.

### CODE REVIEW

**Comment by jbteves** [comment]
@bendhouseart what do these 48 checks do?

**Comment by bendhouseart** [comment]
> @bendhouseart what do these 48 checks do?

Too much!!! Fixed it to only run on mac and ubuntu for python versions 3.7, 3.8, and 3.9. It breaks on python 3.6 on mac, so I was trying to get around it.

**Comment by bendhouseart** [comment]
I can make it fancier/more selective, but right now it installs and runs the tests and passes on most use cases, plus there should be a neat little status badge on the new readme.

**Comment by bendhouseart** [comment]
To break this down -> ` Run Tests / Setup_and_Run_Tests (ubuntu-latest, 3.7) (pull_request) Successful in 30s`

This specific workflow ran `Run Tests` -> `Setup_and_Run_Tests` -> on ubuntu latest, python version `3.7` and succeeded.

Since there are 2 Operation systems, 3 versions of python to test, and the actions run on Push and Pull request  (2) we  get a total of 2x3x2=12 actions/workflow runs.

Normally on push it will be just 6, but b/c this is a PR it runs some more. Could possibly be elimnated since a PR requires a push, but ehh...

**Comment by jbteves** [comment]
You could go ahead and add 3.10 to the mix, people are starting to use it and since I only use builtins except for the testing suite, the dependency management should be light on that front.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Archivotron
+
+[![Run Tests](https://github.com/jbteves/archivotron/actions/workflows/archivotron_actions.yaml/badge.svg)](https://github.com/jbteves/archivotron/actions/workflows/archivotron_actions.yaml)
+
+Here to solve all of the worlds problems.
```
Comment (jbteves) [inline README.md]:
```suggestion
Here to solve some of the world's problems.
```

File: .gitignore [hunk 2]
```diff
@@ -140,3 +140,4 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+.idea/
```
Comment (jbteves) [inline .gitignore]:
```suggestion
```

**Review by jbteves — APPROVED** [review]
LGTM

**Pull request merged**